Fibonacci.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+public class Fibonacci {
+  public static void main(String[] args) {
+    int i = 0;
+    double FibNumber = 0;
+    int MaxNumb = 2147483647;
+    double sqrt5 = Math.sqrt(5);
+    double GoldenRatio = (1 + sqrt5) / 2;
+    Scanner input = new Scanner(System.in);
+    System.out.println("Enter a positive number to 2147483647: ");
+    int number = input.nextInt();
+    if (number >= 0 && number <= MaxNumb) {
+      while (FibNumber < number) {
+        FibNumber = Math.round(Math.pow(GoldenRatio, i) / sqrt5);

```

Comment:
This is not reliable way as it gives approximate value. As you didn't ask if it's ok to have effective but not precise solution this is not acceptable